internal/acceptance/openstack/objectstorage/v1/objects_test.go

```diff
@@ -330,7 +330,6 @@ func TestObjectsListSubdir(t *testing.T) {
 	}
 
 	listOpts = objects.ListOpts{
-		Full:      true,
```

Comment:
List option `Full: false` (`accept: text/plain`) doesn't work and should not be used. We always list "full" now (`accept: application/json`).